sys/include/phydat.h

```diff
@@ -98,6 +98,7 @@ enum {
     /* pressure */
     UNIT_BAR,       /**< Beer? */
     UNIT_PA,        /**< Pascal */
+    UNIT_HPA,       /**< hectoPascal */
```

Comment:
I wasn't that confident about it either. The rational behind this was, that currently we can only have values like `1013hPa` (still needs this PR for `h` prefix). But its not possible to retain high resolution and do something like `1013.3hPa` -> which would imply `UNIT_HPA` and `scale = -1`